api/bundles.testdata.php

```diff
@@ -0,0 +1,8 @@
+<?php
+  require_once('bundles.util.php');
+
+  header('Content-Type: application/json; charset=utf-8');
+  header('Cache-Control: max-age=0');
+  $result = scanKeyboardDirs();
+  echo json_encode($result, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
+?>
```

Comment:
Is this file used in a test? If so, would it belong in the `./test` folder?

Reply:
It can be used to generate the initial test data files -- which I then edited to make them shorter. So it's not used in a test.